### PR TITLE
Remove Gemfile.lock, update Dependabot, fix broken links.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Remove `Gemfile.lock`; it doesn't actually affect what we run in production and we waste a bunch of time applying Dependabot updates.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
A bunch of link fixups needed.

File: Gemfile.lock [hunk 1]
```diff
@@ -1,297 +0,0 @@
...
-    unf (0.1.4)
-      unf_ext
-    unf_ext (0.0.8)
-    unicode-display_width (1.8.0)
-    yell (2.2.2)
-    zeitwerk (2.5.4)
-
-PLATFORMS
-  ruby
-
```
Comment (zakwarlord7) [inline Gemfile.lock:290]: deno.xml

File: Gemfile.lock [hunk 2]
```diff
@@ -1,297 +0,0 @@
...
-DEPENDENCIES
-  github-pages
-  html-proofer
-  rake
-
```
Comment (zakwarlord7) [inline Gemfile.lock:295]: spackage-on: Python JS

Comment (zakwarlord7) [reply Gemfile.lock:295]: -with: Aconda

**Review by zakwarlord7 — COMMENTED** [review]
**Approve**
Squash_mrg: 
[bandos cheatplate.jpng.xlsx](https://github.com/github/opensource.guide/files/9211967/bandos.cheatplate.jpng.xlsx)
)

**Pull request merged**